starter-code/scripts/articleView.js

```diff
@@ -73,41 +73,69 @@ articleView.setTeasers = function() {
   });
 };
 
-articleView.initNewArticlePage = function() {
- // TODO: Make the tabs work. Right now, you're seeing all the tab content (items with a class of tab-content) on the page at once. The section with the id of "write" should show when the "write" tab is clicked; it is also the default and should be shown on page load. The section with the id of "articles" should show when the "preview" tab is clicked.
+function Entry() {
```

Comment:
This follows the exact same format as our original Article constructor. Do we want to create a new one here? Or can we use the old code and just create a new Article?